web/static/help/Messaging_es.md

```diff
@@ -0,0 +1 @@
+../../../doc/help/Messaging_es.md
```

Comment:
hmm.....not sure but its showing web/static/help/Messaging.md → web/static/help/Messaging_en.md renamed